src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -1,12 +1,16 @@
+from __future__ import annotations
+
 import datetime
 import re
 import typing
 from collections import defaultdict
+from dataclasses import dataclass, field
 from typing import Any, Dict, Mapping, Optional, Sequence, Tuple, Union
 
 from cloudevents.http import CloudEvent
 from dateutil.parser import parse
 from pydantic import BaseModel, ConfigDict
+from qtpy.QtGui import QColor
```

Comment:
@berland Any strong feelings about importing GUI classes outside of the GUI?
The alternative is changing the type to `Any`. I tried putting the class somewhere else, but that lead to circular import problems.
Maybe a better solution would be to move all the new snapshot/gui dataclasses to a `dataclasses.py` file?

Reply:
If it is a 3-tuple of `int`s, we can call it that. At least not use QColor if it is not such a type.